package.json

```diff
@@ -60,5 +60,8 @@
       "path": "@commitlint/cz-commitlint"
     }
   },
-  "packageManager": "yarn@4.1.0+sha256.81a00df816059803e6b5148acf03ce313cad36b7f6e5af6efa040a15981a6ffb"
+  "packageManager": "yarn@4.1.0+sha256.81a00df816059803e6b5148acf03ce313cad36b7f6e5af6efa040a15981a6ffb",
+  "dependencies": {
+    "dotenv": "^16.4.5"
+  }
```

Comment:
Did you try to add this in devDependencies? Or maybe we can have it in every example-app package.json?

Reply:
I brew-installed this. I'll try to remove since I _think_ we already have it